config/configuration.json

```diff
@@ -540,6 +540,8 @@
     }
   },
   "InvalidTurnRestrictionCheck": {
+    "straight.on.angle.threshold": 60.0,
```

Comment:
Just curious, how were these angles determined for striaght on and uturn?

Reply:
I borrowed the calculation in Osmose and modified it a little bit.

https://github.com/osm-fr/osmose-backend/blob/master/analysers/analyser_osmosis_relation_restriction.py#L384

WHERE
    (type IN ('only_straight_on', 'no_straight_on') AND abs(a) < 180-60) OR
    (type IN ('only_left_turn', 'no_left_turn') AND (a > 0 OR a < -170)) OR
    (type IN ('only_right_turn', 'no_right_turn') AND (a < 0 OR a > 170)) OR
    (type IN ('no_u_turn') AND abs(a) > 100)
   
Yes, the angles are tricky so I may have to adjust depending on FPR analysis.

Reply:
Sounds good! I have also been trying to determine the best angles for this kind of application, specifically for classifying links. I am interested to see what you come up with. Thanks!

Reply:
Yes, waiting for FPR...

Thanks for reviewing!